src/components/taskListContainer/index.tsx

```diff
@@ -1,24 +1,31 @@
+import axios from 'axios';
 import React, { useState, useEffect } from 'react';
-import axios from "axios";
+import { sendRequestToApi } from '../../utility/api';
 import NewTaskInputForm from '../newTaskInputForm';
 import TaskItem from '../taskItem';
-import styles from './styles.module.css'
-import { getAllTasksFromApi } from "./api"
+import styles from './styles.module.css';
+
+type Task = {
+    status: string,
```

Comment:
I set this originally as 
`status: "open" | "closed";` but kept getting weird errors, maybe we can look at this together.

Reply:
no problem